### DISCUSSION TITLE
In Strimizi Operator, could not delete topic from UI

### DISCUSSION THREAD

**Discussion by barryzhounb** [strimzi/strimzi-kafka-operator#4661] [General]
Under OpenShift, we have installed Strimizi Operator, then created Kafka cluster, topic and user, everything worked well.

Then from Strimizi Operator UI, we delete topic, after 1~2 seconds, topic comes back, it seems we could not delete topic. However when we can delete all resources including Kafka cluster, topic and user, it worked.

So, when we only delete topic resource, seem Operator will get topic back. For my purpose, we want to delete a topic, we don't want Operator get topic back, how to do that? Is there any configurations for preventing deleting topic back? Or is there any command to do it?

**Comment by scholzj** [comment]
Most probably that means something is creating the topic again. For example clients through auto-creation if you didn't disabled it or maybe you are deleting one of the internal topics.

**Reply by barryzhounb** [reply]
Yeah, I saw auto.create.topics.enable, by default it is true.

**Reply by barryzhounb** [reply]
Thanks
